doc/rados/configuration/common.rst

```diff
@@ -74,22 +77,23 @@ For additional details, see the `Monitor Config Reference`_.
 
 .. _ceph-osd-config:
 
-
 Authentication
 ==============
 
 .. versionadded:: Bobtail 0.56
 
-For Bobtail (v 0.56) and beyond, you should expressly enable or disable
-authentication in the ``[global]`` section of your Ceph configuration file.
+For Bobtail (v0.56) and later releases, you should expressly enable or disable
```

Comment:
Maybe it's time we assume that the reader isn't running Argonaut or earlier.

Reply:
I know, I know.